ui/src/custom.scss

```diff
@@ -295,3 +295,7 @@ a {
 .moreinfo-tooltip > .tooltip-inner {
   max-width: none;
 }
+
+.small-font {
```

Comment:
I already copied in the tailwind text sizing utility classes (see above). Can you use `text-sm` (or `text-xs`) instead of defining a new one?